apps/schools/domains/ticket/components/currentTicket/index.tsx

```diff
@@ -0,0 +1,276 @@
+                    <span>{is_internal_recipient ? 'Внутренний комментарий' : null}</span>
+                </div>
+                <div className={styles.comment}>{comment}</div>
+            </div>
+        </div>
+    )
+}
+
+const CurrentTicket = () => {
+    const [text, setText] = useState<string>('')
+    const [currentStatus, setCurrentStatus] = useState<string>('')
+    const [pageTime, setPageTime] = useState(0)
+    const [isInternal, setIsInternal] = useState(false)
+    const [isButtonDisabled, setIsButtonDisabled] = useState(false)
+    const [currentDependencies, setCurrentDependencies] = useState<string[]>([])
+
+    const { organizationId } = useOrganization()
+    const { user } = useUserProfile()
+    const uuid = getUuidFromUrl()
+
+    const [mutation] = useChangeStatusMutation()
+    const [mutationTicket] = useSeenTicketCommentMutation()
+    const [mutationCreateComment] = useCreateCommentMutation()
+
+    const { data: ticketData, isLoading: isTicketLoading } = useGetTicketQuery({
+        organization_id: organizationId,
+        ticket_id: uuid[0],
+    })
+
+    const { data: ticketComments, isLoading: isCommentsLoading } = useGetTicketCommentsQuery({
+        ticket_id: uuid[0],
+    })
+
+    const { data, isLoading } = useGetAllEmployeesQuery({
+        organization: organizationId,
+        employee_profile: user.employee_profile?.id,
+    })
+
+    const SendComment = () => {
+        setIsButtonDisabled(true)
+        handleCreateComment(uuid[0], mutationCreateComment, text, data?.results[0].id, isInternal).then((success) => {
+            if (success) {
+                setIsInternal(false)
+                setText('')
+                handleChange
+            }
+            setIsButtonDisabled(false)
+        })
+    }
+
+    const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+        const { value } = event.target
+        setText(value)
+    }
+
+    const calculateHeight = (text: string): string => {
+        const lineCount = text.split('\n').length
+        const charCount = text.length - text.split('\n').length
+        const extraHeight = Math.floor(charCount / 67) * 22.5
+
+        return `${lineCount * 22.5 + extraHeight}px`
+    }
+
+    const statusTranslations: { [key: string]: { translate: string; color: string } } = {
+        SENT: {
+            translate: 'Новое',
+            color: SENT_FILTER_COLOR,
+        },
+        IN_PROGRESS: {
+            translate: 'Открыто',
+            color: IN_PROGRESS_FILTER_COLOR,
+        },
+        CLOSED: {
+            translate: 'Закрыто',
+            color: CLOSED_FILTER_COLOR,
+        },
+    }
+
+    const translateStatus = (status: string) => {
+        return statusTranslations[status]?.translate || status
+    }
+
+    const decorateStatus = (status: string) => {
+        return statusTranslations[status]?.color || '#fff'
+    }
+
+    const graph: { [key: string]: string[] } = {
```

Comment:
I would change transformed value to initial values from StatusesEnum
It would be better